services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/commonutils/CommonUtils.java

```diff
@@ -0,0 +1,161 @@
+package com.sourcefuse.jarc.services.usertenantservice.commonutils;
+
+import com.sourcefuse.jarc.services.usertenantservice.DTO.Group;
+import com.sourcefuse.jarc.services.usertenantservice.DTO.Role;
+import com.sourcefuse.jarc.services.usertenantservice.DTO.Tenant;
+import com.sourcefuse.jarc.services.usertenantservice.DTO.UserTenant;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CommonUtils<T> {
+
+    public void copyProperties(T source, T destination) {
```

Comment:
Have already suggested a better method please remove this

Reply:
done